README.md

```diff
@@ -8,6 +8,7 @@ Collection of blogs/publications by Nepali devs.
 Name | Tags
 ------------ | -------------
 [Ajay Gautam](https://medium.com/@evilboyajay) | cyber security
+[Aju Tamang](https://www.csaju.com) | python, system design, programming general
```

Comment:
@Aju100 I didn't find any system design related blogs on your site, can you update the tag?